REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.6
 MathProgBase 0.6 0.8
-ReverseDiffSparse 0.8 0.9
```

Comment:
Please keep the upper bound. I'll tag the next ReverseDiffSparse release as 0.8.x.